app/javascript/components/search/controls/download/DownloadCommand.js

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect, useRef } from 'react'
 
 import LoadingSpinner from 'lib/LoadingSpinner'
 import { fetchAuthCode, stringifyQuery } from 'lib/scp-api'
+import { _ } from '@databiosphere/bard-client/src/utils'
```

Comment:
so that this doesn't get confused with lodash
```suggestion
import { _ as bardUtils } from '@databiosphere/bard-client/src/utils'
```